scaleapi/__init__.py

```diff
@@ -2,24 +2,19 @@
 
 from .tasks import Task
 from .batches import Batch
+from .projects import Project
 
 TASK_TYPES = [
-    'annotation',
-    'audiotranscription',
     'categorization',
-    'comparison',
-    'cuboidannotation',
-    'datacollection',
     'imageannotation',
-    'lineannotation',
     'namedentityrecognition',
-    'pointannotation',
-    'polygonannotation',
     'segmentannotation',
-    'transcription',
+    'documenttranscription',
     'videoannotation',
-    'videoboxannotation',
-    'videocuboidannotation'
+    'videoplaybackannotation',
+    'namedentityrecognition',
+    'textcollection',
+    'documentmodel'
 ]
```

Comment:
@nofel-scale you're actually missing a big part of why this array exists. 

At the bottom of __init__.py, we have:
```
def _AddTaskTypeCreator(task_type):
    def create_task_wrapper(self, **kwargs):
        return self.create_task(task_type, **kwargs)
    setattr(ScaleClient, 'create_' + task_type + '_task', create_task_wrapper)
for taskType in TASK_TYPES:
    _AddTaskTypeCreator(taskType)
```

This allows the client to have `client.create_imageannotation_task` as a method, plus all the other task types. You can still use the raw `create_task` method for things like Lidar if you wanted, but we should call out in the readme that these other functions do exist. (You removed it for now)